test.js

```diff
@@ -63,9 +71,13 @@ test('async with concurrency: 2 (out of order time sequence)', async t => {
 });
 
 test('enforce number in options.concurrency', async t => {
-	await t.throws(m([], () => {}, {concurrency: 0}), TypeError);
-	await t.throws(m([], () => {}, {concurrency: undefined}), TypeError);
-	await t.notThrows(m([], () => {}, {concurrency: 1}));
-	await t.notThrows(m([], () => {}, {concurrency: 10}));
-	await t.notThrows(m([], () => {}, {concurrency: Infinity}));
+	await t.throwsAsync(() => m([], () => {}, {concurrency: 0}), TypeError);
+	await t.throwsAsync(() => m([], () => {}, {concurrency: undefined}), TypeError);
+	await t.notThrows(() => m([], () => {}, {concurrency: 1}));
+	await t.notThrows(() => m([], () => {}, {concurrency: 10}));
+	await t.notThrows(() => m([], () => {}, {concurrency: Infinity}));
```

Comment:
This is not a correct change.
```suggestion
	await t.notThrowsAsync(m([], () => {}, {concurrency: Infinity}));
```

Same with the other ones.